src/Resolve/graphtype.jl

```diff
@@ -432,7 +432,7 @@ function _add_fixed!(graph::Graph, fixed::Dict{UUID,Fixed})
     vdict = graph.data.vdict
 
     for (fp,fx) in fixed
-        haskey(pdict, fp) || error("unknown fixed package $(pkgID(fp, graph))")
+        haskey(pdict, fp) || error("unknown fixed package $(logstr_pkgID(fp, graph))")
```

Comment:
This isn't a log message this is an error so probably should not change it 

```suggestion
        haskey(pdict, fp) || error("unknown fixed package $(pkgID(fp, graph))")
```